app/models/server/models/LivechatInquiry.js

```diff
@@ -6,7 +6,6 @@ export class LivechatInquiry extends Base {
 
 		this.tryEnsureIndex({ rid: 1 }); // room id corresponding to this inquiry
 		this.tryEnsureIndex({ name: 1 }); // name of the inquiry (client name for now)
-		this.tryEnsureIndex({ message: 1 }); // message sent by the client
```

Comment:
@sampaiodiego do we heavily rely on this index? If yes, we can consider hashed or text indexes. Else, removing the index entirely seems like a good plan 🤔

Reply:
looks like we don't use that index, so removing it would not be a big deal. but I'd like to have @renatobecker eyes on this as well.

but why removing the index fix the issue?

Reply:
@sampaiodiego mongodb has index limits for 4.0 or later (https://docs.mongodb.com/manual/reference/limits/#index-limitations). For `message` field max length is 1024 byte. If the message will be more than 1024 byte, mongodb can't insert this document. For mongodb 4.2 or greater index key limits removed

Reply:
I didn't find where the index is using. Also I didn't find where the `message` field is using

Reply:
thanks @antkaz for poiting it out, I didn't see that on a first glance.. 

for this case I miss database transactions, as if something fail during "room creating process", everything should have been undone and an error displayed.. but anyways, thanks for the fix.. let's just wait @renatobecker confirms this is the proper solution